docs/src/main/asciidoc/security-basic-authentication-tutorial.adoc

```diff
@@ -1,11 +1,13 @@
 [id="security-basic-authentication-tutorial"]
 = Secure a Quarkus application with Basic authentication and JPA
 include::_attributes.adoc[]
-:categories: security,getting-started
+:categories: security,getting-started,jpa
```

Comment:
Is there a category set up in the docs architecture for `jpa`?

We don't mention one in the doc contributor guide and I don't see it in the drop-down list on the doc portal yet.